python/analyzers/HNLGenTreeAnalyzer.py

```diff
@@ -32,6 +32,8 @@ def beginLoop(self, setup):
         self.counters.addCounter('HNLGenTree')
         count = self.counters.counter('HNLGenTree')
         count.register('all events')
+        count.register('bad event: failed saving prompt lepton')
+        count.register('bad event: failed saving neutrino')
```

Comment:
what are these for?

Reply:
they're not being called anymore. I removed them.